css/materialize.css

```diff
@@ -8583,6 +8583,7 @@ a.waves-effect .waves-ripple {
   padding: 0 12px;
   border-radius: 16px;
   background-color: #e4e4e4;
+  margin: 10px 0px;
```

Comment:
When we have a someone's library we are not changing things in it. That is for we have our own css where we can overwrite some particular pieces  that are suitable for us. Can you move it to `styles.css` file, please. 
`.chip{
 margin: 10px 0px;
}`